.github/workflows/ci.yml

```diff
@@ -0,0 +1,100 @@
+# © 2024. TU Dortmund University,
+# Institute of Energy Systems, Energy Efficiency and Energy Economics,
+# Research group Distribution grid planning and operation
+#
+
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - dev
+      - 'feature/*'
+      - 'hotfix/*'
+      - 'release/*'
+  pull_request:
+    branches:
+      - main
+      - dev
```

Comment:
It's generally considered best practice to keep the pull_request trigger, as it can help catch potential edge cases. In the end, it doesn’t hurt to have it in place.